09-01-24/I-like-animals/js-unit-testing_get-animal/index.js

```diff
@@ -1 +1,9 @@
-export function getAnimal() {}
+export function getAnimal(animal) {
+  return animal === 'cats'
```

Comment:
you should start with an `if` conditional. 
`if (animal === "cats")

Reply:
it depending of the syntax, the "?" if..
 does it work to you?